src/inverseRobot.js

```diff
@@ -23,8 +23,21 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const inverseObj = {};
```

Comment:
You should make it maximally clear. Let's say 
```suggestion
  const inversedRobot = {};
```